include/openrave/kinbody.h

```diff
@@ -38,6 +38,12 @@ enum GeometryType {
     GT_Cage=6, ///< a container shaped geometry with removable side walls. The side walls can be on any of the four sides. The origin is at the bottom of the base. The inner volume of the cage is measured from the base to the highest wall.
 };
 
+enum DynamicsConstraints {
```

Comment:
what about ``DynamicsConstraintsType``?

Reply:
I was going to ask the exact same question. Since it's an `enum` of type/option for torque checking, I think it might be more informative/less confusing if the name is something like `DynamicsConstraintsType`.

Reply:
@Puttichai @kanbouchou i i agree, let's make it consistent